htdocs/install/mysql/tables/llx_societe_rib.sql

```diff
@@ -23,6 +23,10 @@
 create table llx_societe_rib
 (
   rowid          integer AUTO_INCREMENT PRIMARY KEY,
+
+  --For Stripe or other external payment method
+  ref varchar(128),								-- 'card_...', 'src_...' or 'pm_...'
```

Comment:
"ref" is reserved for internal dolibarr ref.
Can you use
"ext_payment_mode_id"
instead ?

Reply:
@eldy  i need it for fetch() in class/companypaymentmode.class.php it use dolibarr fetchcommon class.

So we need to code a new fetch ?

Reply:
Yes. The common fetch can be used when we need a fetch on internal id or internal ref.

So, if we need a fetch on a field that is not an internal id or ref, it means we must have a specific implementation for the fetch
fetch($id, $ref=null, $ext_payment_mode_id=0)
(The 2 parameters must always be id and ref, even if there is no ref, then we can add specific parameters).

Reply:
Can you enhance the fetch and rename "ref" into "ext_payment_mode_id" ?